.github/workflows/tests.yaml

```diff
@@ -58,7 +58,7 @@ jobs:
   unit-tests:
     strategy:
       matrix:
-        os: [ubuntu-20.04, ubuntu-22.04, ubuntu-24.04]
+        os: [ubuntu-22.04, ubuntu-24.04]
```

Comment:
While we obviously can't do anything about the runners being shut off, I'm mildly concerned that we may now introduce bugs on the still-supported focal, especially with plugin updates. Obviously we're not running in exactly the same environment that craft-parts runs in within an app, but this was for example how I found out that focal doesn't have a `poetry` package.

It's a non-blocker here, but we should consider whether we want to ask for some self-hosted focal runners to use.

Reply:
Agreed, it would be nice to do some further testing on older systems.  I also agree that we can do this in a follow up effort (possibly driven by starflow).